sn_cli/src/subcommands/files.rs

```diff
@@ -93,6 +93,9 @@ async fn upload_files(
     // The input files_path has to be a dir
     let file_names_path = root_dir.join("uploaded_files");
 
+    // The temp dir for stream encryptor to flush encrypted chunks.
```

Comment:
Just a small Q, why do we want to write the encrypted chunks to file? Sorry might be a dumb question.

Reply:
We do this right now so that we can pay for them all in one go. 

It mayyy work out that we just one one tx per chunk. It's more to validate but may be fine. (Although if we're needing the change dbc to pay for the next it could work out really llooooong).

Something to investigate down the line